ksso/aws.py

```diff
@@ -0,0 +1,127 @@
+    shell environment variable exports.
+
+    This function is intended to output the necessary `export` commands for
+    AWS credentials, allowing users to quickly set their environment variables
+    for temporary access to AWS resources. The output can be used directly
+    in a Bash shell by sourcing the command.
+
+    Args:
+        credentials (Dict[str, str]): A dictionary containing AWS temporary
+                                      credentials, which must include:
+                                      - 'AccessKeyId': The AWS access key ID.
+                                      - 'SecretAccessKey': The AWS secret key.
+                                      - 'SessionToken': The AWS session token.
+                                      - 'Expiration': The expiration timestamp
+                                        (must support `isoformat()`).
+
+    Example Usage:
+        This function is typically used in the following way to simplify
+        credential management. Example usage in a Bash shell:
+
+            source <(ksso --env \
+                --client-id client-name-aws \
+                --aws-role-arn arn:aws:iam::account_id:role/some-role-name)
+
+    Output:
+        Prints environment variable export commands, which can be directly
+        sourced into the shell to configure AWS credentials temporarily
+    """
+    print(f'export AWS_ACCESS_KEY_ID={credentials["AccessKeyId"]}')
+    print(f'export AWS_SECRET_ACCESS_KEY={credentials["SecretAccessKey"]}')
+    print(f'export AWS_SESSION_TOKEN={credentials["SessionToken"]}')
+    print(f'export AWS_CREDENTIALS_EXPIRATION={credentials["Expiration"].isoformat()}')
+
+
+def export_aws_credentials_json(credentials):
+    """
+    Generate and print AWS temporary credentials in a json format.
+
+    This function is intended to output the json object with valid
+    aws short term credentials, that can be quickly assumed by tools
+    like aws-vault supporting credentials helpers.
+
+    Args:
+        credentials (Dict[str, str]): A dictionary containing AWS temporary
+                                      credentials, which must include:
+                                      - 'AccessKeyId': The AWS access key ID.
+                                      - 'SecretAccessKey': The AWS secret key.
+                                      - 'SessionToken': The AWS session token.
+                                      - 'Expiration': The expiration timestamp
+                                        (must support `isoformat()`).
+
+    Example Usage:
+        This function is typically used in conjunction with a tool like `aws-vault`
+        to simplify credential management. See expected aws-vault configuration
+        in the documentation.
+
+    Output:
+        Prints json object, which can be directly sourced into the shell
+        or cli tools like `aws-vault` supporting credential helpers
+    """
+    aws_credentials = {
```

Comment:
LEt's add a separate class for AWS credentials data at the top of this file pls and use it as an class instances everywhere

Reply:
Done